blockchain/serviceMetadata.go

```diff
@@ -96,6 +97,7 @@ func getMetaDataUrifromRegistry() []byte {
 			WithField("ServiceName", config.GetString(config.ServiceName)).
 			Panic("Error Retrieving contract details for the Given Organization and Service Name ")
 	}
+	defer ethClient.Close()
```

Comment:
Should be right after ```GetEthereumClient()``` call at line 84. Otherwise there is a chance that client will not be closed if for instance ```NewRegistryCaller``` returns error.

Reply:
my bad , fixed this now